bin/pywps_daemon.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python3
+
+from daemon import DaemonContext
```

Comment:
requirement for [python-daemon](https://pypi.org/project/python-daemon/) is missing.

Reply:
done